erpnext/templates/includes/cart.js

```diff
@@ -132,28 +134,30 @@ $.extend(shopping_cart, {
 		var quot_name = name || quotation_name;
 		if (quot_name && quot_name !== undefined && quot_name !== "None") {
 			var a = document.getElementsByClassName("indicator-link")[0];
-			a.href = "/quotations/" + encodeURIComponent(quot_name);
+			a.href = "/purchase-orders/" + encodeURIComponent(quot_name);
 			$('.quotation-name').html("("+quot_name+")");
 			$('.cart-indicator').show();
 		} else {
 			$('.cart-indicator').hide();
 		}
 	},
 
-	bind_change_delivery_date: function() {
+	bind_change_delivery_date: function(quotation_name) {
 		var delivery_date = shopping_cart.field_group.get_value('delivery_date') || "";
+		var quotation_name = $('.indicator-link').attr('data-quotation-name') || "";
```

Comment:
Why are you duplicating `quotation_name`. There is one in argument and one as a local variable